docs/sources/reference/components/loki/loki.source.kubernetes.md

```diff
@@ -163,17 +158,15 @@ Clustering only looks at the following labels for determining the shard key:
 
 ## Exported fields
 
-`loki.source.kubernetes` does not export any fields.
+`loki.source.kubernetes` doesn't export any fields.
```

Comment:
I am a bit surprised we prefer contractions!

Reply:
We generally follow the Google Developer's style guide (with a few modifications as documented in the Writer's Toolkit).  Contractions are preferred as outlined here: https://developers.google.com/style/contractions 

Quoting the guideline: "In general, we write our documentation in an informal [tone](https://developers.google.com/style/tone), so we recommend using most types of contractions."